src/mbgl/util/color.cpp

```diff
@@ -17,4 +18,12 @@ optional<Color> Color::parse(const std::string& s) {
     }};
 }
 
+std::string Color::stringify() const {
+    return "rgba(" +
```

Comment:
Named colors will also come back as `rgba()` strings; there are no guarantees that values are round-tripped exactly. This is consistent with GL JS and the CSSOM.